runtime/rasdump/javadump.cpp

```diff
@@ -1052,14 +1052,31 @@ JavaCoreDumpWriter::writeEnvironmentSection(void)
 	/* Write the running mode data */
 	_OutputStream.writeCharacters("1CIRUNNINGAS   Running as ");
 
+	/* If JITServer enabled, print running mode as JITServer and client UID*/
+#ifdef JITSERVER_SUPPORT
```

Comment:
Note that we could have JITSERVER_SUPPORT enabled and still be running as a regular JVM (when we don't specify options to run as a client or server)

Reply:
How should we use `clientUID` to differentiate each case? There are three conditions here:
- JITSERVER_SUPPORT enabled, running as server -> clientUID = 0
- JITSERVER_SUPPORT enabled, running as client -> clientUID = non-zero random number
- JITSERVER_SUPPORT enabled, running as regular JVM

I propose to set clientUID to 1 as a flag of regular JVM. So 0 means jitserver, 1 means regular JVM, other means client.

Reply:
`if (vm->internalVMFunctions->isJITServerEnabled(vm))`  will tell you if the JVM is a server. If it's not, then check jitConfig->clientUID. If 0, it's a regular JVM, othwerwise it's a client